moc-health-check/README.md

```diff
@@ -0,0 +1,66 @@
+# MOC Health Reporter - v1.0
+
+********
+***Overview***
+********
+
+This repository provides a real-world application for pfurl project. It sends a generic command from pfurl to send local files to Pfioh, run a sample job on Pman, and fetch the files back from Pfioh. Overall, it tests all of the core elements of Chris Platform, informing personnel if an issue pertains for a long peridod of time. 
```

Comment:
The purpose doesn't really seem like it's to add real world application.

Reply:
I would use lowercase for pman and pfioh consistently.

Reply:
s/Chris/ChRIS

Reply:
I would remove the line about informing personnel since that shouldn't be part of this script.  In general I would rewrite this paragraph to match what we have talked about.